packages/web-components/src/select/select.spec.md

```diff
@@ -0,0 +1,100 @@
+# Select
+
+## Component Description
+
+The browser-native select control component allows users to choose one option from a dropdown list. It is commonly used in forms for selecting values from a predefined set of options.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/0vOH481cv1VQyfQIX9JALn/Select?node-id=1319-163&t=nn2xgT4ieHjbkqQW-11)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr appearance: "outline" "underline" "filled-darker" "filled-lighter" | "outline"
+- @attr size: "small" "medium" "large" | "medium"
+- @attr disabled: boolean
+- @attr multiple: boolean
+- @attr required: boolean
+- @attr name: string
+- @attr autofocus: boolean
+- @attr autocomplete: "on" "off" | "off"
+- @attr aria-label: string
+- @attr aria-labelledby: string
+- @attr aria-describedby: string
+
+### Outputs
+
+None
+
+### Events
+
+- @input: Fires whenever the value of the `select` has been changed
+- @change: Fires whenever the user modifies the value of `select`
+
+### Slots
+
+- default: child elements such as '<option>' will be placed in the default slot
+- icon: the icon, typically a down arrow
+
+### CSS Variables
+
+None
+
+## Accessibility
+
+- [x] Find the matching component through WCAG's patterns: https://www.w3.org/WAI/ARIA/apg/patterns/
+  - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select
+- [x] Are there any accessibility elements unique to this component?
+- [x] List ARIA attributes
+  - @attr aria-label: string
+  - @attr aria-labelledby: string
+  - @attr aria-describedby: string
+- [x] Does the component support 400% zoom?
+- [x] What keyboard behaviors does the component support?
+  - Up / Down : Moves focus between options
+  - Space / Enter: Selects the focused option
+  - Home : Sets focus to the first option
+  - End : Sets focus to the last option
+  - Esc : Closes the dropdown list if it is open
+  - Page Up / Down : Moves focus between options
+
+## Preparation
+
+- [x] [Find the base FAST Component](https://explore.fast.design/components/) this component will inherit from and document
+  - This will be a wrapper for the browser-native select control and does not have a corresponding FAST component
+- [x] [Check the Fluent UI React V9 Component Spec](https://github.com/microsoft/fluentui/tree/master/specs) for differences and document
+  - No spec available
```

Comment:
We should probably document any deltas with the react component absent the spec when we merge the component work.